src/witan/plot/chart.clj

```diff
@@ -0,0 +1,96 @@
+(ns witan.plot.chart
+  (:require [cljplot.build :as plotb]
+            [cljplot.config :as cfg]
+            [cljplot.core :as plot]
+            [cljplot.render :as plotr]
+            [clojure.string :as s]
+            [witan.plot.colors :as colors]
+            [witan.plot.series :as ps]))
+
+(defn zero-index-numerical-y-axes [prepped-data]
+  (let [[t [_bottom top]] (get-in prepped-data [:extents :y 0])]
+    (if (= :numerical t)
+      (plotb/update-scale prepped-data :y :domain [0 (+ (* 0.025 top) top)])
```

Comment:
Magic number (0.025)?

Reply:
it is a bit magic number atm, as it is "enough, given what I've seen so far".